scripts/maps.js

```diff
@@ -249,3 +249,64 @@ function synchronizeMapsKey(keyObj, current, previous) {/*
         updateSingleMapCheck(check, true);
     }*/
 }
+
+function getAvailableChecks(check) {
+    $(check ? (typeof(check) == typeof(1) ? "[data-requirements-glitchless*='" + check + "'],[data-requirements-glitchless*='" + (check - 1) + "'],[data-requirements-glitchless*='" + (check + 1) + "']" : "[data-requirements-glitchless*=\"" + check + "\"]") : "[data-requirements-glitchless]").each(function() {
+        var reqs = JSON.parse('[' + $(this).attr('data-requirements-glitchless').replaceAll("'", '"').replaceAll('\\"', "'") + ']');
+        var available = true;
+        for (var req of reqs) {

```

Comment:
Can you comment this chunk? I can generally tell what it's doing based on the html changes, but it would be nice to have at least an example and how it's broken down here as well.

Reply:
I added some comments. Does that help clear things up?

Reply:
perfect thanks